Description: fix the perl interpreter path to the one typically found on Debian
Author: gregor hermann <gregoa@debian.org>
Forwarded: not-needed
Last-Update: 2010-11-19
Reviewed-by: Nicholas Bamber <nicholas@periapt.co.uk>

--- a/eg/startup.pl
+++ b/eg/startup.pl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 # to load this file when the server starts, add this to httpd.conf:
 # PerlRequire /path/to/startup.pl
